framework/src/base/Assembly.C

```diff
@@ -1547,3 +1547,9 @@ Assembly::clearCachedJacobianContributions()
   _cached_jacobian_contribution_cols.reserve(1.2*orig_size);
   _cached_jacobian_contribution_vals.reserve(1.2*orig_size);
 }
+
+bool
+Assembly::isBlockDiagonal()
```

Comment:
Hmm, I think this might have been necessary before the warehouse system was changed. But yea now I'm not using it, so it can go!